lib/plugins/static.js

```diff
@@ -44,8 +44,13 @@ function serveStatic(options) {
 
     function serveFileFromStats(file, err, stats, isGzip, req, res, next) {
         if (err) {
-            next(new ResourceNotFoundError(err, '%s', req.path()));
-            return;
+            var defaultFile = path.join(opts.directory, opts.default);
```

Comment:
Can you add `opts.default` to the assertion block at the top to ensure we get a proper format for the input?